certified-connectors/Asite/Readme.md

```diff
@@ -0,0 +1,32 @@
+# About
+
+Asite is an open construction platform that enables organizations with comprehensive range of solutions connect dispersed teams across the lifecycle of capital assets by collaborate, plan, design, and build with seamless information sharing across the entire supply chain which helps capital project owners stay at the forefront of innovation, maintaining a golden thread of information.
+
+The Asite connector helps to build a connection between two systems for file exchange. The connector collaborates both systems by uploading and downloading files based on triggers.
+
+# Pre-requisites
+
+- Required Asite CDE Subscription
+- Active Asite login credentials
+- Configuration on Asite for required workflow trigger of type &#39;Microsoft Flow&#39;
```

Comment:
@asitedeveloper Please let us know what is `&#39` in above text. Please note that unicodes are not allowed.